lib/gen_stage.ex

```diff
@@ -1039,18 +1044,7 @@ defmodule GenStage do
 
   ## Options
 
-    * `:name` - used for name registration as described in the "Name
-      registration" section of the module documentation
-
-    * `:timeout` - if present, the server is allowed to spend the given amount of
-      milliseconds initializing or it will be terminated and the start function
-      will return `{:error, :timeout}`
-
-    * `:debug` - if present, the corresponding function in the [`:sys`
-      module](http://www.erlang.org/doc/man/sys.html) is invoked
-
-    * `:spawn_opt` - if present, its value is passed as options to the
-      underlying process as in `Process.spawn/4`
+  This function accepts the same options as `GenServer.start_link/3`.
```

Comment:
Let's list `name` and `debug` and ask them to look at GenServer for other options then.